### PR TITLE
Release/1.3.0

### LINKED ISSUE #347: Release version 1.3.0
### Describe your question

This issue is for tracking changes for the 1.3.0 release.  Target release date: **28 October 2022**.

### Pre-release steps
- n/a

### [Release steps](https://github.com/globeandmail/sophi-for-wordpress/blob/develop/CONTRIBUTING.md#release-instructions)
- [x] Branch: Starting from `develop`, cut a release branch named `release/1.3.0` for your changes.
- [x] Version bump: Bump the version number in `sophi.php`, `package.json`, and `readme.txt` if it does not already reflect the version being released.  Update both the plugin "Version:" property and the plugin `SOPHI_WP_VERSION` constant in `sophi.php`.
- [x] Changelog: Add/update the changelog in `CHANGELOG.md` and `readme.txt`.
- [x] Props: Update `CREDITS.md` with any new contributors, confirm maintainers are accurate.
- [x] Readme updates: Make any other readme changes as necessary.  `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content.  The two are slightly different.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.distignore`.
- [x] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the Pull Request), then do the same for `develop` into `trunk` (`git checkout trunk && git merge --no-ff develop`).  `trunk` contains the stable development version.
- [ ] Test: While still on the `trunk` branch, test for functionality locally.
- [x] Push: Push your `trunk` branch to GitHub (e.g. `git push origin trunk`).
- [x] Release: Create a [new release](https://github.com/globeandmail/sophi-for-wordpress/releases/new), naming the tag and the release with the new version number, and **targeting the `trunk` branch**.  Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the [closed issues on the milestone](https://github.com/globeandmail/sophi-for-wordpress/milestone/13?closed=1).  The release should now appear under [releases](https://github.com/globeandmail/sophi-for-wordpress/releases).
- [x] SVN: Wait for the [GitHub Action](https://github.com/globeandmail/sophi-for-wordpress/actions/workflows/dotorg-push-deploy.yml) to finish deploying to the WordPress.org repository. If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [x] Check WordPress.org: Ensure that the changes are live on https://wordpress.org/plugins/sophi/. This may take a few minutes.
- [x] Close milestone: Edit the [milestone](https://github.com/globeandmail/sophi-for-wordpress/milestone/13) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description field`), then close the milestone.
- [x] Punt incomplete items: If any open issues or PRs which were milestoned for `1.3.0` do not make it into the release, update their milestone to `1.4.0` or `Future Release`.
- [x] Version bump (again): In the `develop` branch (`cd ../ && git checkout develop`) bump the version number in `sophi.php` to `1.3.1-dev`.  It's okay if the next release might be a different version number; that change can be handled right before release in the first step, as might also be the case with `@since` annotations.
- [x] Test sites: Check the Sophi test sites to ensure the new version has been deployed there.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by jeffpaul**
### Description of the Change

Release and version bump to 1.3.0.

Closes #347.

### Alternate Designs

None

### Possible Drawbacks

None

### Verification Process

Verified testing via VS Code and the GitHub Desktop UI

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests passed.

### Changelog Entry

n/a

### Credits

n/a

### CODE REVIEW

**Comment by jeffpaul** [comment]
@faisal-alvi note that I need confirmation on the changelog entry from #333 and the correct name to display for the Override hook.

File: CHANGELOG.md
Comment (dkotter) [inline CHANGELOG.md]:
```suggestion
- New tutorial for modifying tracking data on documentation site (props [@iamdharmesh](https://github.com/iamdharmesh), [@YMufleh](https://github.com/YMufleh), [@dkotter](https://github.com/dkotter) via [#327](https://github.com/globeandmail/sophi-for-wordpress/pull/327)).
```

File: readme.txt
Comment (dkotter) [inline readme.txt]:
```suggestion
* **Added:** New tutorial for modifying tracking data on documentation site (props [@iamdharmesh](https://github.com/iamdharmesh), [@YMufleh](https://github.com/YMufleh), [@dkotter](https://github.com/dkotter) via [#327](https://github.com/globeandmail/sophi-for-wordpress/pull/327)).
```

File: CHANGELOG.md
Comment (dkotter) [inline CHANGELOG.md]:
```suggestion
- Replaced hardcoded tracker endpoint value with the new Tracker Address setting (props [@faisal-alvi](https://github.com/faisal-alvi), [@YMufleh](https://github.com/YMufleh), [@jeffpaul](https://github.com/jeffpaul) via [#342](https://github.com/globeandmail/sophi-for-wordpress/pull/342)).
```

File: readme.txt
Comment (dkotter) [inline readme.txt]:
```suggestion
* **Changed:** Replaced hardcoded tracker endpoint value with the new Tracker Address setting (props [@faisal-alvi](https://github.com/faisal-alvi), [@YMufleh](https://github.com/YMufleh), [@jeffpaul](https://github.com/jeffpaul) via [#342](https://github.com/globeandmail/sophi-for-wordpress/pull/342)).
```

**Review by dkotter — DISMISSED** [review]

**Comment by faisal-alvi** [comment]
@jeffpaul the changelog entry for #333, something like:

Introduced 2 new WP REST Endpoints.
1. [GET] `/wp-json/sophi/v1/site-automation` - To Get the Curator Posts.
2. [POST] `/wp-json/sophi/v1/site-automation-override` - To override the curator Posts.

---

And if we want to add only for the override one, we can simply add something like:

The new `/wp-json/sophi/v1/site-automation-override` REST Endpoint to allow for code-based Override feature integrations.

**Review by faisal-alvi — APPROVED** [review]

**Comment by jeffpaul** [comment]
Note that I'm holding off on shipping the release until we get approval from the Sophi team that testing on their end has passed and that general plugin functionality and the Overrides feature is working as expected.

**Pull request merged**
